diff --git a/modules/search/search.extender.inc b/modules/search/search.extender.inc
index ad4b86e..bad3402 100644
--- a/modules/search/search.extender.inc
+++ b/modules/search/search.extender.inc
@@ -272,7 +272,7 @@ class SearchQuery extends SelectQueryExtender {
         foreach ($key as $or) {
           list($num_new_scores) = $this->parseWord($or);
           $any |= $num_new_scores;
-          $queryor->condition('d.data', "% $or %", 'LIKE');
+          $queryor->condition('d.data', "%$or%", 'LIKE');
         }
         if (count($queryor)) {
           $this->conditions->condition($queryor);
@@ -284,7 +284,7 @@ class SearchQuery extends SelectQueryExtender {
       else {
         $simple_and = TRUE;
         list($num_new_scores, $num_valid_words) = $this->parseWord($key);
-        $this->conditions->condition('d.data', "% $key %", 'LIKE');
+          $this->conditions->condition('d.data', "%$key%", 'LIKE');
         if (!$num_valid_words) {
           $this->simple = FALSE;
         }
@@ -297,7 +297,7 @@ class SearchQuery extends SelectQueryExtender {
     }
     // Negative matches.
     foreach ($this->keys['negative'] as $key) {
-      $this->conditions->condition('d.data', "% $key %", 'NOT LIKE');
+      $this->conditions->condition('d.data', "%$key%", 'NOT LIKE');
       $this->simple = FALSE;
     }
 
@@ -353,7 +353,7 @@ class SearchQuery extends SelectQueryExtender {
     if (!empty($this->words)) {
       $or = db_or();
       foreach ($this->words as $word) {
-        $or->condition('i.word', $word);
+        $or->condition('i.word', "%$word%", 'LIKE');
       }
       $this->condition($or);
     }
